### ISSUE TITLE
An error was reported when I installed it

### ISSUE THREAD

**Issue by dfgao** [theMILOlab/SPATA2#156]
Dear developer:

An error was reported when I installed it: **obsUnit.Rd:25: Bad \link option -- must be text**
How can I fix it? thx

My sessioninfo:

R version 4.3.3 (2024-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /usr/lib64/libopenblasp-r0.3.3.so;  LAPACK version 3.8.0

locale:
 [1] LC_CTYPE=zh_CN.UTF-8       LC_NUMERIC=C               LC_TIME=zh_CN.UTF-8        LC_COLLATE=zh_CN.UTF-8    
 [5] LC_MONETARY=zh_CN.UTF-8    LC_MESSAGES=zh_CN.UTF-8    LC_PAPER=zh_CN.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=zh_CN.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods   base   

other attached packages:
 [1] hypeR_2.0.0                 data.table_1.15.4           ggsci_3.1.0                
 [4] plotly_4.10.4               future_1.33.2               lubridate_1.9.3            
 [7] forcats_1.0.0               stringr_1.5.1               dplyr_1.1.4                
[10] purrr_1.0.2                 readr_2.1.5                 tidyr_1.3.1                
[13] tibble_3.2.1                ggplot2_3.5.1               tidyverse_2.0.0            
[16] rio_1.1.1                   SoupX_1.6.2                 scDblFinder_1.16.0         
[19] SingleCellExperiment_1.24.0 SummarizedExperiment_1.32.0 Biobase_2.62.0             
[22] GenomicRanges_1.54.1        GenomeInfoDb_1.38.8         IRanges_2.36.0             
[25] S4Vectors_0.40.2            BiocGenerics_0.48.1         MatrixGenerics_1.14.0      
[28] matrixStats_1.3.0           BPCells_0.2.0               Seurat_5.1.0               
[31] SeuratObject_5.0.2          sp_2.1-4

**Comment by kueckelj** [comment]
This error occured during installation with `install_github()`?

**Comment by dfgao** [comment]
> 安装过程中出现此错误`install_github()`？

yes, I followed this link to install it : https://themilolab.github.io/SPATA2/articles/installation.html

**Comment by kueckelj** [comment]
I am sorry but I can not reproduce this error. Can you restart your R session, make sure that the environment is clear and no packages except for baseR packages are attached, especially not SPATA2, and try again?

**Comment by dfgao** [comment]
> I am sorry but I can not reproduce this error. Can you restart your R session, make sure that the environment is clear and no packages except for baseR packages are attached, especially not SPATA2, and try again?

I will try again as soon as possible and give feedback. Thank you very much.

**Comment by satatri** [comment]
I had the same issue when I tried to install it. The error message is related to the documentation file of obsUnit line 25: 

`Extracts the observational unit of the \link[\code{SpatialMethod}]{spatial method (platform)}`

**Comment by kueckelj** [comment]
Hello! Unfortunately, I cannot reproduce the error. How did you install it exactly? And can you try with `devtools::install_github("theMILOLab/SPATA2", "dev")`? Does that also result in an error?

**Comment by satatri** [comment]
Thank you for your swift response. I tried to install it with `devtools::install_github(repo = "theMILOlab/SPATA2")`. I tried again like you suggested but I had the same error message: "obsUnit.Rd:25: Bad \link option -- must be text".

**Comment by kueckelj** [comment]
`devtools::install_github("theMILOLab/SPATA2", "dev")` includes "dev"! this installs the dev branch. Did you use that?

**Comment by satatri** [comment]
Yes I tried using `devtools::install_github("theMILOLab/SPATA2", "dev")` like you suggested but it results in the same error for me

**Comment by kueckelj** [comment]
This is weird. We can not reproduce this. What is your current R version and Rstudio version?

**Comment by satatri** [comment]
R version: 4.4.0 (2024-04-24)
RStudio version: 2024.4.2.764.1
OS: Rocky Linux 8.10 (Green Obsidian)
